### PR TITLE
picocom: update to 2024-07

### PR DESCRIPTION
**Pull request by BC204**
Topic Description
-----------------

- picocom: build documentation picocom.1
- picocom: update to 2024-07

Package(s) Affected
-------------------

- picocom: 1:2024-07

Security Update?
----------------

No

Build Order
-----------

```
#buildit picocom
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-utils/picocom/autobuild/build [hunk 1]
```diff
@@ -1,4 +1,5 @@
 make
```
Comment (MingcongBai) [inline app-utils/picocom/autobuild/build:2]: Lint this script please - abinfo, verbosity, etc.

File: app-utils/picocom/spec [hunk 2]
```diff
@@ -1,4 +1,4 @@
-VER=3.1
-SRCS="tbl::https://github.com/npat-efault/picocom/archive/$VER.tar.gz"
-CHKSUMS="sha256::e6761ca932ffc6d09bd6b11ff018bdaf70b287ce518b3282d29e0270e88420bb"
+VER=2024-07
```
Comment (MingcongBai) [inline app-utils/picocom/spec]: This should be:

```bash
UPSTREAM_VER=2024-07
VER="${UPSTREAM_VER//\-/.}"
```

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-utils/picocom/autobuild/build (hunk 1 above)
Comment (BC204) [reply app-utils/picocom/autobuild/build:1]: The upstream removed the independent file picocom.1 in the commit [c66d82fd](https://gitlab.com/wsakernel/picocom/-/commit/c66d82fd958d027fd48a6253e8b017653df1b190).
However, **make (all)** cannot generate the manpage picocom.1 and **make doc** needs to be executed additionally.
https://gitlab.com/wsakernel/picocom/-/blob/master/Makefile?ref_type=heads#L77

Comment (BC204) [reply app-utils/picocom/autobuild/build:1]: 截图 2024-08-22 17-27-44

**Pull request closed without merging**